src/app/order/OrderConfirmation.tsx

```diff
@@ -136,7 +139,7 @@ class OrderConfirmation extends Component<
                             />
                         </OrderConfirmationSection> }
 
-                        { this.renderGuestSignUp({
+                        { isAccountCreationEnabled && this.renderGuestSignUp({
```

Comment:
Should the API return `customerCanBeCreated` as `false` if account creation is disabled for the store?

Reply:
Done https://github.com/bigcommerce/bigcommerce/pull/39522/files#diff-cf023e67e771e1df05463d400e87bf828174f00eaedaaf379a92fbdb0a6c4d38R106